python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -514,3 +545,15 @@ cdef Column _update_column_struct_field_names(
             )
         col.set_base_children(tuple(children))
     return col
+
+cdef _get_col_children_names(Column col, column_in_metadata& col_meta):
```

Comment:
```suggestion
cdef _set_col_children_names(Column col, column_in_metadata& col_meta):
```

I think this can be named as a set method since we are not returning any result.